crates/rpc/rpc/src/eth/api/server.rs

```diff
@@ -298,10 +298,10 @@ where
         return Ok(EthApi::gas_price(self).await?)
     }
 
-    /// Handler for: `eth_blobGasPrice`
+    /// Handler for: `eth_blobBaseFee`
     async fn blob_gas_price(&self) -> Result<U256> {
```

Comment:
let's also change this method name in the trait

Reply:
Done